static/redfish/v1/JsonSchemas/OemServiceRoot/index.json

```diff
@@ -0,0 +1,65 @@
+    "definitions": {
+        "IBM": {
+            "additionalProperties": false,
+            "description": "OEM Extension for IBM Manager Information",
+            "longDescription": "OEM Extension for IBM Manager Information including date, time, Model number, Serial number.",
+            "patternProperties": {
+                "^([a-zA-Z_][a-zA-Z0-9_]*)?@(odata|Redfish|Message)\\.[a-zA-Z_][a-zA-Z0-9_]*$": {
+                    "description": "This property shall specify a valid odata or Redfish property.",
+                    "type": [
+                        "null",
+                        "string"
+                    ]
+                }
+            },
+            "properties": {
+                "Model": {
+                    "description": "The product name for this system, without the manufacturer name.",
+                    "longDescription": "This property shall describe how the manufacturer refers to this system.  Typically, this value is the product name for this system without the manufacturer name.",
+                    "readonly": true,
+                    "type": [
+                        "string",
+                        "null"
+                    ]
+                },
+                "SerialNumber": {
+                    "description": "The serial number for this system.",
+                    "longDescription": "This property shall contain the serial number for the system.",
+                    "readonly": true,
+                    "type": [
+                        "string",
+                        "null"
+                    ]
+                },
+                "DateTime": {
+                    "description": "The current date and time with UTC offset of the manager.",
+                    "format": "date-time",
+                    "longDescription": "This property shall contain the current date and time with UTC offset of the manager.",
+                    "readonly": false,
+                    "type": [
+                        "string",
+                        "null"
+                    ]
+                },
+                "DateTimeLocalOffset": {
+                    "description": "The time offset from UTC that the DateTime property is in `+HH:MM` format.",
+                    "longDescription": "This property shall contain the offset from UTC time that the DateTime property contains.  If both DateTime and DateTimeLocalOffset are provided in modification requests, services shall apply DateTimeLocalOffset after DateTime is applied.",
+                    "pattern": "^([-+][0-1][0-9]:[0-5][0-9])$",
+                    "readonly": false,
+                    "type": [
+                        "string",
+                        "null"
+                    ]
+                }
+            },
+            "type": "object"
+        }
+    },
+    "owningEntity": "IBM",
+    "release": "1.0",
+    "title": "#OemServiceRoot.v1_0_0"
```

Comment:
At https://github.com/ibm-openbmc/bmcweb/pull/130/files this doesn't have a version

Reply:
Done.